mon-pix/tests/integration/components/routes/campaigns/restricted/join-sup-test.js

```diff
@@ -24,6 +24,20 @@ describe('Integration | Component | routes/campaigns/restricted/join-sup', funct
   });
 
   context('when the student number is typed', () => {
+
+    it('should focus the student number input', async function() {
+      // given
+      this.set('onSubmitToReconcileStub', onSubmitToReconcileStub);
```

Comment:
Question : 
Est-ce que `this.onSubmitToReconcileStub = onSubmitToReconcileStub;`suffirait ?

Reply:
Yes, je l'ai mis en BSR en remplaçant tous les .set du fichier test par un acces direct.